### PR TITLE
Add option in dashboard cli for AWS vpc and subnet

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
### Description

This PR adds options in dashboard cli for cloud deployment.  Originally, those options were in server and client cloud deployment scripts.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/dashboard/cli.py [hunk 1]
```diff
@@ -192,6 +196,18 @@ def define_dashboard_parser(parser):
...
+        "--vpc-id",
+        type=str,
+        default="",
+        help="VPC id for AWS EC2 instance.  Applicable to AWS only.  Ignored if subnet-id is not specified.",
+    )
+    parser.add_argument(
+        "--subnet-id",
+        type=str,
+        default="",
+        help="Subnet id for AWS EC2 instance.  Applicable to AWS only.  Ignored if pvc-id is not specified.",
```
Comment (YuanTingHsieh) [inline nvflare/dashboard/cli.py:209]: pvc or vpc?

Comment (IsaacYangSLA) [reply nvflare/dashboard/cli.py]: Same typo from the another PR.  Fixed both.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Pull request merged**